Gemfile

```diff
@@ -61,6 +61,7 @@ gem 'will_paginate', '>= 3.0.6'
 gem 'will_paginate-bootstrap', '>= 1.0.1'
 gem 'georuby', '2.0'
 gem 'geokit-rails'
+gem 'json'
```

Comment:
Oops, seeing this Gemfile addition here too. Do you really think this is needed? If so, we could accept it but I think the code is running OK without this gem, isn't it?